content/sigs/cloud-native/index.adoc

```diff
@@ -153,6 +157,12 @@ Participant links will be posted in the link:https://gitter.im/jenkinsci/configu
 
 When needed the topics discussed at Office Hours meeting will be reported back at Cloud Native SIG meeting.
 
+=== Jenkins Kubernetes Operator
+
+Jenkins Kubernetes Operator can realize the automatic operation and maintenance of Jenkins in the kubernetes cluster.
+
+The Jenkins Operator will integrate with the CasC plugin, Groovy init scripts and more to enable configuration control、 stateful control and other possible improvements of Jenkins in the kubernetes cluster.
```

Comment:
Maybe just integration with https://github.com/jenkinsci/custom-war-packager then?

Reply:
In looking at the custom-war-packager I see possible use of that but not integration with that.
I feel the operator should be standalone